### PR TITLE
Kratos Dev Setup + Client Config

### PR DESCRIPTION
**Pull request by valentinyanakiev**

### CODE REVIEW

File: .build/ory/kratos/email-password/kratos.yml [hunk 1]
```diff
@@ -4,50 +4,63 @@ dsn: memory
...
   methods:
     password:
       enabled: true
+    oidc:
+      enabled: true
+      config:
+        providers:
+          - id: microsoft
+            provider: microsoft
+            client_id: feb6d8b1-8cc2-4e7d-a419-ad7b544b0832
```
Comment (techsmyth) [inline .build/ory/kratos/email-password/kratos.yml:27]: presume this is from our playground env?

File: .env.docker [hunk 2]
```diff
@@ -11,12 +11,15 @@ AUTH_AAD_LOGGING_PII=false
 AUTH_AAD_LOGGING_LEVEL=Error
 AUTH_AAD_UPN_DOMAIN=playgroundcherrytwist.onmicrosoft.com
```
Comment (techsmyth) [inline .env.docker:12]: The aad / demo auth ones can also go given the config is removed?

File: quickstart-server.yml [hunk 3]
```diff
@@ -34,6 +34,8 @@ services:
       - IPFS_ENDPOINT
       - IPFS_CLIENT_ENDPOINT
       - IPFS_MAX_FILE_SIZE
+      - AUTH_ORY_KRATOS_ISSUER
```
Comment (techsmyth) [inline quickstart-server.yml:24]: Good while modifying this file to also remove the AAD / DEMO AUTH env vars

File: quickstart-services.yml [hunk 4]
```diff
@@ -2,6 +2,13 @@ version: '3.7'
 
 networks:
   ct_server_dev:
+  ct_dev_net:
+    driver: bridge
+    ipam:
+      driver: default
+      config:
+        - subnet: 192.168.10.0/24
```
Comment (techsmyth) [inline quickstart-services.yml:10]: hardcoded?

File: quickstart-services.yml [hunk 5]
```diff
@@ -82,37 +89,24 @@ services:
...
-      - PORT=4435
-      - KRATOS_BROWSER_URL=http://127.0.0.1:4455/.ory/kratos/public
-      - JWKS_URL=http://oathkeeper:4456/.well-known/jwks.json
-      - SECURITY_MODE=jwks
-    networks:
-      - ct_server_dev
-    restart: on-failure
+      - ct_dev_net
 
   mailslurper:
```
Comment (techsmyth) [inline quickstart-services.yml:81]: surely this can go?

**Review by techsmyth — APPROVED** [review]
Great set of updates, love how many things are taken out. 

Some comments / questions mostly - but am ok for this to go in if client is nearly ready to work with it.

File: .build/ory/kratos/email-password/kratos.yml (hunk 1 above)
Comment (valentinyanakiev) [reply .build/ory/kratos/email-password/kratos.yml:27]: yes, of course, all is playground config

File: quickstart-services.yml (hunk 4 above)
Comment (valentinyanakiev) [reply quickstart-services.yml:10]: Yes, it is a private bridge interface shared between docker and the host.

File: quickstart-server.yml (hunk 3 above)
Comment (valentinyanakiev) [reply quickstart-server.yml:24]: Fixed in f47154b44e7560df4e8aae338ccfb66e68d84677

**Pull request merged**